frameworks/C++/lithium/lithium.cc

```diff
@@ -89,49 +100,51 @@ int main(int argc, char* argv[]) {
   };
 
   my_api.get("/updates") = [&](http_request& request, http_response& response) {
+    set_max_sql_connections_per_thread(2);
     std::string N_str = request.get_parameters(s::N = std::optional<std::string>()).N.value_or("1");
     int N = atoi(N_str.c_str());
     N = std::max(1, std::min(N, 500));
+    std::vector<decltype(random_numbers.all_fields())> numbers(N);
 
-     
-    auto c = random_numbers.connect(request.yield);
-    auto& raw_c = c.backend_connection();
+    {     
+      auto c = random_numbers.connect(request.yield);
+      auto& raw_c = c.backend_connection();
 
-    std::vector<decltype(random_numbers.all_fields())> numbers(N);
-    
+      
 #if TFB_MYSQL
-    raw_c("START TRANSACTION");
+      raw_c("START TRANSACTION");
 #endif
-    for (int i = 0; i < N; i++)
-    {
-      numbers[i] = c.find_one(s::id = 1 + rand() % 9999).value();
-      numbers[i].randomNumber = 1 + rand() % 9999;
-    }
+      for (int i = 0; i < N; i++)
+      {
+        numbers[i] = c.find_one(s::id = 1 + rand() % 9999).value();
+        numbers[i].randomNumber = 1 + rand() % 9999;
```

Comment:
Looks like I missed this before

`1 + rand() % 9999` -> 1 to 9999

Could you update to `1 + rand() % 10000`